api/src/app.ts

```diff
@@ -1,11 +1,12 @@
 import express, { Request, Response, NextFunction } from "express";
-import { fetchProducts } from "./logic";
+import { fetchProducts, login } from "./logic";
 import { PORT } from "./config";
 
 const app = express();
 const cors = require("cors");
 
 app.use(cors());
+app.use(express.json());
 
 app.get("/", (req: Request, res: Response) => {
```

Comment:
Please, remove the Hello world endpoint.
You can expose one endpoint named /healthcheck which returns 200 OK, does not need authentication and returns some message like - "API is running...". In the usual cases we expose such endpoint that is called by Load Balancer or EKS Cluster probe check. This way we are able to determine the API is up and running.